app/views/ncr/work_orders/form.html.erb

```diff
@@ -2,10 +2,10 @@
   <h4>National Capital Region</h4>
   <h3>Purchase Card Approval Request</h3>
 
-  <%= form_for @proposal_form, url: @form_url, method: @form_method do |f| %>
+  <%= form_for @work_order do |f| %>
     <div class="form-group">
-      <%= f.label :description %>
-      <%= f.text_area :description, class: 'form-control' %>
+      <%= label_tag :description %>
+      <%= text_area_tag :description, @description, class: 'form-control' %>
```

Comment:
This got swallowed in an outdated diff:

Do you think description should not depend on cart? I can definitely add it as its own field and add an indirection layer as with `total_amount`